code/game/objects/effects/spawners/random/exotic.dm

```diff
@@ -48,67 +48,29 @@
 		/obj/item/storage/medkit/brute = 27,
 		/obj/item/storage/medkit/fire = 27,
-		/obj/item/grenade/clusterbuster/smoke = 15,
-		/obj/item/clothing/under/chameleon = 13,
 		/obj/item/storage/toolbox/syndicate = 12,
-		/obj/item/melee/baton/telescopic = 12,
-		/obj/item/melee/baton = 11,
-		/obj/item/book/granter/spell/smoke = 10,
-		/obj/item/book/granter/spell/blind = 10,
-		/obj/item/clothing/shoes/chameleon/noslip = 10,
-		/obj/item/grenade/c4 = 7,
 		/obj/item/borg/upgrade/ddrill = 3,
-		/obj/item/borg/upgrade/soh = 3,
+		/obj/item/knife/butcher = 14,
+		/obj/item/clothing/glasses/night = 10,
+		/obj/item/pickaxe/drill/diamonddrill = 6,
 	)
 
 /obj/effect/spawner/random/exotic/antag_gear
 	name = "antag gear"
 	icon_state = "esword"
 	loot = list(
-		/obj/item/storage/medkit/tactical = 35,
-		/obj/item/book/granter/spell/summonitem = 20,
-		/obj/item/book/granter/spell/forcewall = 17,
-		/obj/item/pneumatic_cannon = 15,
-		/obj/item/book/granter/spell/knock = 15,
-		/obj/item/storage/backpack/holding = 12,
-		/obj/item/shield/riot/tele = 12,
-		/obj/item/stack/sheet/mineral/diamond{amount = 15} = 10,
-		/obj/item/stack/sheet/mineral/uranium{amount = 15} = 10,
-		/obj/item/stack/sheet/mineral/plasma{amount = 15} = 10,
-		/obj/item/stack/sheet/mineral/gold{amount = 15} = 10,
-		/obj/item/grenade/spawnergrenade/spesscarp = 7,
-		/obj/item/melee/energy/sword = 7,
-		/obj/item/borg/upgrade/disablercooler = 7,
-		/obj/item/dnainjector/lasereyesmut = 7,
-		/obj/item/shield/energy = 6,
-		/obj/item/pickaxe/drill/diamonddrill = 6,
-		/obj/item/grenade/spawnergrenade/manhacks = 6,
-		/obj/item/defibrillator/compact = 6,
-		/obj/item/book/granter/spell/barnyard = 4,
-		/obj/item/grenade/clusterbuster/inferno = 3,
-		/obj/item/gun/magic/wand/fireball/inert = 3,
-	)
 
-/obj/effect/spawner/random/exotic/antag_gear_strong
-	name = "antag gear strong"
-	icon_state = "esword_dual"
-	loot = list(
-		/obj/item/pickaxe/drill/jackhammer = 30,
-		/obj/item/singularityhammer = 25,
-		/obj/item/fireaxe = 25,
-		/obj/item/organ/brain/alien = 17,
-		/obj/item/borg/upgrade/selfrepair = 17,
-		/obj/item/gun/ballistic/automatic/c20r/unrestricted = 16,
-		/obj/item/dualsaber = 15,
-		/obj/item/gun/magic/wand/resurrection/inert = 15,
-		/obj/item/grenade/clusterbuster/spawner_manhacks = 15,
-		/obj/item/borg/upgrade/syndicate = 13,
-		/obj/item/book/granter/spell/charge = 12,
-		/obj/item/book/granter/spell/fireball = 10,
-		/obj/item/gun/magic/wand/resurrection = 10,
-		/obj/item/mjollnir = 10,
-		/obj/item/organ/heart/demon = 7,
-		/obj/item/uplink/old = 2,
+		/obj/item/clothing/glasses/science/night = 15,
+		/obj/item/shield/riot = 12,
+		/obj/item/stack/sheet/mineral/diamond{amount = 15} = 5,
+		/obj/item/stack/sheet/mineral/uranium{amount = 15} = 5,
+		/obj/item/stack/sheet/mineral/plasma{amount = 15} = 5,
+		/obj/item/stack/sheet/mineral/gold{amount = 15} = 5,
+		/obj/item/grenade/clusterbuster/smoke = 15,
+		/obj/item/clothing/under/chameleon = 13,
```

Comment:
chameleon jumpsuits really aren't that exciting without other chameleon gear to go with them; they generally just get concealed by your other worn equipment